src/main/kotlin/test/ktortemplate/core/utils/json/JsonSettings.kt

```diff
@@ -0,0 +1,23 @@
+package test.ktortemplate.core.utils.json
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+object JsonSettings {
+
+    val mapper = Json {
+        ignoreUnknownKeys = true
+        prettyPrint = true
+    }
+
```

Comment:
Maybe move these two helper functions into the test scope since these are only used there.

Reply:
I think @aanciaes  did the right thing here, as this ensures that we use a consistent set of mapper options for our enoding/decoding purposes.